src/EFCore.SqlServer/Scaffolding/Internal/SqlServerDatabaseModelFactory.cs

```diff
@@ -1335,13 +1348,16 @@ private void GetTriggers(DbConnection connection, IReadOnlyList<DatabaseTable> t
     }
 
     private bool SupportsTemporalTable()
-        => _compatibilityLevel >= 130 && _engineEdition != 6;
+        => _compatibilityLevel >= 130 && (_engineEdition != 6 && _engineEdition != 11 && _engineEdition != 1000);
```

Comment:
nit (embrace the new!):

```suggestion
        => _compatibilityLevel >= 130 && _engineEdition is not 6 and not 11 and not 1000;
```